cfr.yaml

```diff
@@ -65,7 +65,7 @@
       author: Juan Escobar
       icon: resource://activity-stream/data/content/assets/cfr_google_translate.png
       id: '445852'
-      rating: 4.2
+      rating: "4.2"
```

Comment:
Let's piggyback with a rating update. It's "4.1" now. https://addons.mozilla.org/en-US/firefox/addon/to-google-translate/?src=search